tests/foreman/api/test_ansible.py

```diff
@@ -602,3 +602,102 @@ def test_negative_ansible_job_timeout_to_kill(
         assert [i['output'] for i in result if i['output'] == termination_msg]
         assert [i['output'] for i in result if i['output'] == 'StandardError: Job execution failed']
         assert [i['output'] for i in result if i['output'] == 'Exit status: 120']
+
+    @pytest.mark.tier2
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_match('8')
+    @pytest.mark.parametrize('setting_update', ['remote_execution_effective_user'], indirect=True)
```

Comment:
It won't be required if we handle effective user at job level, right?

Reply:
I am handling it by updating the settings page. The JobInvocation() method does not have an option to update the effective user.

Reply:
you can add it in the job invocation method then

Reply:
"I can add that, but as mentioned above, it doesn't produce valid output.

Reply:
As discussed and as per BZ, we can skip setting effective_user to run this playbook